.circleci/config.yml

```diff
@@ -9,27 +9,37 @@ jobs:
         environment:
           - GO111MODULE=on
           - GOLANG_LINT_VERSION=v1.27.0
+          - OPERATOR_SDK_VERSION=v0.18.2
     working_directory: /go/src/github.com/kiegroup/kogito-cloud-operator
     steps:
       - checkout
+      - run:
+          name: Install operator-sdk
+          command: |
+            sudo curl -L https://github.com/operator-framework/operator-sdk/releases/download/$OPERATOR_SDK_VERSION/operator-sdk-$OPERATOR_SDK_VERSION-x86_64-linux-gnu -o /usr/bin/operator-sdk
+            sudo chmod +x /usr/bin/operator-sdk
+            # For debugging
+            operator-sdk version
+      - run:
+          name: Check Vet
+          command: |
+            ./hack/go-vet.sh
+            changed_files=$(git status -s | grep -v 'go.mod\|go.sum' || :)
+            openapi_changed_files=$(git status -s | grep zz_generated || :)
```

Comment:
Just a nitpick in here. But `zz_generated` is also a prefix for the controller-gen object deepcopy generation. So the variable name could be `generated_changed_files`. :D

Reply:
What do you suggest should we handle only the openapi files or for both generated files?

Reply:
I think both. DeepCopy also changes if have a `*_types` change. Your script already will caught that, just need to change the var name.

Reply:
Updated